pyproject.toml

```
@@ -10,7 +10,7 @@ packages = [{include = "async_processor"}]
 # TODO (chiragjn): Loosen these requirements for use as a library dependency. Sidecar method can choose to have exactly pinned versions
 python = ">=3.8,<3.13"
 prometheus_client = ">=0.17.1,<0.18.0"
-fastapi = ">=0.100.1,<0.101.0"
+fastapi = ">=0.100.1,<=0.104.1"
```

Can we allow lower versions? say
```suggestion
fastapi = ">=0.90.0,<=0.104.1"
```